examples/app-npm-webpack/messages/ru.json

```diff
@@ -0,0 +1,18 @@
+{
+	"ru": {
+		"intro-1": "Используйте Globalize для интернационализиции вашего приложения.",
+		"number-label": "Число",
+		"currency-label": "Валюта",
+		"date-label": "Дата",
+		"relative-time-label": "Относительное время",
+		"message-1": "Пример сообщения со смешанными числами \"{number}\", валютой \"{currency}\", датами \"{date}\" и относительным временем \"{relativeTime}\".",
+		"message-2": [
+			"Пример сообщения с поддержкой множественного числа:",
+			"{count, plural,",
+			"   one {У вас осталась одна задача}",
+			"   many {У вас осталось # задач}",
```

Comment:
Well, it might be. But I'd rather put in all the classes, this is an example and it should not contain ambiguities or unclear defaults.